### PR TITLE
Update customcontroller.Harmony.xml

### PR DESCRIPTION
**Pull request by glorious1**
Corrected typos and removed some redundant assignments

## Description
Corrected typos and removed redundancies as follows (original line numbers)
42             <button id="102">Down</button>
Although it's just a comment, 'minus' should probably be 'down'.

47            <button id="107">Fullscreen</button>
I don't know if action names are case sensitive, but elsewhere it is 'FullScreen'.

108            <button id="185">xbmc.ActivateWindow(SystemSettings)</button>
Should 'xbmc.' be dropped?

114            <button id="16">ActivateWindow(PlayerControls)</button>
Button id should be 106

There are quite a few button assignments that are the same between Global and other windows.  If I understand correctly, they don't need to be specified in other windows if they are Global.  Not an error really, but ....  Some example lines are 157, 196, 231, 304-314.

In FullScreenVideo, lines 158-159, those actions are already available in Global, but through different buttons (F10 and F11).

The section ScreenCalibration, lines 235-242, is duplicated in lines 243-249, except that one button is missing in the copy (button id 165).

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: system/keymaps/customcontroller.Harmony.xml [hunk 1]
```diff
@@ -154,9 +154,6 @@
       <!-- right      	-->      <button id="104">StepForward</button>
       <!-- menu       	-->      <button id="106">OSD</button>
       <!-- Prev		-->      <button id="132">SmallStepBack</button>
-      <!-- Info  	-->      <button id="131">Info</button>
-      <!-- F7		-->      <button id="195">NextSubtitle</button>
```
Comment (Memphiz) [inline system/keymaps/customcontroller.Harmony.xml]: In global section the ids 195 and 194 are mapped differently. You can't remove them here bevause you'd loose the subtitle features that are mapped here

File: system/keymaps/customcontroller.Harmony.xml [hunk 2]
```diff
@@ -240,13 +235,6 @@
       <!-- Guide  	-->      <button id="165">NextResolution</button>
     </customcontroller>
   </ScreenCalibration>
-  <ScreenCalibration>
-    <customcontroller name="Harmony">
-      <!-- OK		-->      <button id="105">NextCalibration</button>
```
Comment (Memphiz) [inline system/keymaps/customcontroller.Harmony.xml]: Same here - the global mapping of those buttons is not usefull for the screen calobration. You can't delete them here.

**Review by Memphiz — CHANGES_REQUESTED** [review]
Beside my comments it is fine after requested changes were made

**Comment by Memphiz** [comment]
Thx for digging into github for this. I know its a scary place at the beginning but this is i nice way to get your contribution into Kodi :)

File: system/keymaps/customcontroller.Harmony.xml (hunk 2 above)
Comment (glorious1) [reply system/keymaps/customcontroller.Harmony.xml]: I don't get why there needs to be two ScreenCalibration sections, one after the other, mapping almost the same.  But again, there's a lot I don't understand.

File: system/keymaps/customcontroller.Harmony.xml (hunk 1 above)
Comment (glorious1) [reply system/keymaps/customcontroller.Harmony.xml]: But in global section the subtitle features are mapped to 174-175, which should be still available in the FullScreenVideo section.  I guess I don't understand how it works.

**Comment by Memphiz** [comment]
Hold on - about to board a plane - will have a closer look tomorrow - maybe i misread ...

File: system/keymaps/customcontroller.Harmony.xml (hunk 1 above)
Comment (Memphiz) [reply system/keymaps/customcontroller.Harmony.xml]: I can't tell which buttons 194 and 195 are refering to. I just think that they were mapped for a reason in the first place.

File: system/keymaps/customcontroller.Harmony.xml (hunk 2 above)
Comment (Memphiz) [reply system/keymaps/customcontroller.Harmony.xml]: you are right with this change - i didn't see the second section in the diff ...

**Review by Memphiz — APPROVED** [review]

**Comment by Memphiz** [comment]
fine as is now that i looked into the original file. Thx for the pull request.

**Pull request merged**